package.json

```diff
@@ -51,139 +52,140 @@
   ],
   "author": "Cube Dev Team",
   "peerDependencies": {
-    "react": ">= 17.0.0",
-    "react-dom": ">= 17.0.0",
-    "styled-components": ">= 5.3.0"
+    "react": ">= 18.2.0",
+    "react-dom": ">= 18.2.0",
+    "styled-components": ">= 5.3.6"
```

Comment:
Peer deps should be left as is since they describe the lowest boundary. That's why specifying the patch version isn't necessary and we currently not using any concurrent features in react 18

Reply:
My intention was to support React Server Component which will also require upgrading Styled-Components to v6 (the latest is v6.0.0-beta.9) which is in beta and has some breaking changes.

Reply:
These newer versions of react* are not required to run UI Kit. So there is no need to bump versions just yet.